doc/changelog/04-tactics/17503-redexpr.rst

```diff
@@ -1,6 +1,6 @@
 - **Added:**
-  :tacn:`lazy`, :tacn:`simpl` and :tacn:`cbn` and the associated :cmd:`Eval` and :tacn:`eval` reductions
-  learnt to do head reduction when given flag `head`
+  :tacn:`lazy`, :tacn:`simpl`, :tacn:`cbn` and :tacn:`cbv` and the associated :cmd:`Eval`
+  and :tacn:`eval` reductions learnt to do head reduction when given flag `head`
   (eg `Eval lazy head in (fun x => Some ((fun y => y) x)) 0` produces `Some ((fun y => y) 0)`)
   (`#17503 <https://github.com/coq/coq/pull/17503>`_,
   by Gaëtan Gilbert).
```

Comment:
```suggestion
  (`#18190 <https://github.com/coq/coq/pull/18190>`_,
  by Hugo Herbelin).
```

Also change the file name to reflect 18190.

Reply:
The point is that the feature is provided by the conjunction of two PRs. So, the name of the file can only be chosen arbitrarily. Since there is already a file 17503-redexpr.rst, it seems as easy to keep it (it is anyway an internal name).

In the content of the file however, I agree that #18190 should be mentioned, so it could be:
```
(`#17503 <https://github.com/coq/coq/pull/17503>`_,
  by Gaëtan Gilbert, extended with `#18190 <https://github.com/coq/coq/pull/18190>`_,
  by Hugo Herbelin).
```
or something like that, if @SkySkimmer agrees.

Reply:
yes